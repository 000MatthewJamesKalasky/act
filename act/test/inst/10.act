defproc test()
{
   int x @ [iscomb=0];
   int y @ [iscomb=1];
   
   x = y;
}

test t;
