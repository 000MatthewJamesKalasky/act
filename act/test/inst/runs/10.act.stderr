In expanding test (10.act:1)
In expanding ::<Global>
In merging attributes for x and y
FATAL: Attribute `iscomb': inconsistent values
